### PR TITLE
Jorgegimeno/bump to django 2.2

### LINKED ISSUE #44: Upgrade to Django 2.2
We are currently using Django 1.11 which is no longer supported. We should update.

### PR DESCRIPTION
**Pull request by jlgimeno**
First PR for this project, open to suggestions/comments/concerns.  Thank you!

Closes #44

### CODE REVIEW

File: pythonsd/urls.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
     ),
-    url(
-        r"^meetup-widget\.(?P<format>html|json)$",
+    path(
+        "meetup-widget.<str:format>",
         MeetupWidget.as_view(),
         name="meetup-widget",
     ),
-    url(r"^coc/?$", generic.RedirectView.as_view(url="/pages/code-of-conduct.html")),
+    path("coc", generic.RedirectView.as_view(url="/pages/code-of-conduct.html")),
```
Comment (davidfischer) [inline pythonsd/urls.py:18]: This works on `/coc` but not `/coc/`. I feel like we should fix that (and maybe have a test).

File: config/settings/base.py [hunk 2]
```diff
@@ -54,7 +54,6 @@
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
-    "django.contrib.auth.middleware.SessionAuthenticationMiddleware",
```
Comment (davidfischer) [inline config/settings/base.py:57]: This looks like it was just removed and has no functionality after 1.10. Seems fine.

**Review by davidfischer — COMMENTED** [review]
This looks very close. I tested it without issue except for my comment on `/coc/`.

File: pythonsd/urls.py (hunk 1 above)
Comment (jlgimeno) [reply pythonsd/urls.py:18]: Added a test and a new route.  I felt that was the simplest answer.

**Review by davidfischer — APPROVED** [review]

**Comment by davidfischer** [comment]
This is going to clash with #45 but I want to get it merged. I'll see if I can fix up that one to be compatible with these changes.

**Pull request merged**